package.json

```diff
@@ -61,6 +61,7 @@
     "lint:js-in-markdown": "standard-markdown docs",
     "create-api-json": "electron-docs-linter docs --outfile=electron-api.json",
     "create-typescript-definitions": "npm run create-api-json && electron-typescript-definitions --in=electron-api.json --out=electron.d.ts && node spec/ts-smoke/runner.js",
+    "gn-typescript-definitions-win": "npm run create-typescript-definitions && copy electron.d.ts",
```

Comment:
Instead of duplicating this I'd rather we use something like [`shx`](https://www.npmjs.com/package/shx) as a dev dep to provide `cp` on both platforms

Reply:
thanks your review I got it. I agree your opinion, 
I think, your solution should add in build instructions doc

Reply:
@MarshallOfSound instead of doing the copy here, why not add a gn copy dep https://gn.neocities.org/#copy . It will work across platforms.

Reply:
shx not work in window~ I tried install npm -g shx. after that, I run build electron, but build make same error

Reply:
@deepak1556 this is a strange hack that is required for the TS build.  We went through this when it was initially added. A gn action output must the be in out directory but the TS definitions need to be in the electron directory.  So we gen them in the electron directory and copy them out to trick gn into thinking an actual output item was made

Reply:
@MadfishDT check out the usage instructions for `shx`.  You don't use `cp` you use `shx cp`

Reply:
Is it not possible to do the reverse ? Use the action output in the gen directory and copy them to electron directory by using `copy() {}`

Reply:
It's also against the rules to use GN copy to copy something to a somewhere not in the out directory.  GN doesn't support this use case so we had to hack around it

Reply:
Thanks for the clarification!

Reply:
thanks~ all